### ISSUE TITLE
Importing as the same local name should throw an error on transpile.

### ISSUE THREAD

**Issue by rwjblue** [esperantojs/esperanto#95]
### Example 1:

Input:

```javascript
import { stuff } from 'foo';
import { stuff } from 'bar';

stuff();
```

Output:

```javascript
define(['foo', 'bar'], function (foo, bar) {

	'use strict';

	bar.stuff();

});
```

----

### Example 2

Input:

```javascript
import stuff from 'foo';
import stuff from 'bar';

stuff();
```

Output:

```javascript
define(['foo', 'bar'], function (stuff, stuff) {

	'use strict';

	stuff['default']();

});
```

---

I think we should throw an error at transpile time for both scenarios above.

**Comment by rwjblue** [comment]
This was discovered after updating Ember, and realizing I accidentally had imported with the same name twice (fixed in https://github.com/emberjs/ember.js/pull/10338).

**Comment by Rich-Harris** [comment]
> I think we should throw an error at transpile time for both scenarios above.

Agree. Thanks - fixed

**Comment by Rich-Harris** [comment]
Released in 0.6.16
